### PR TITLE
Plex add media_player server config

### PR DESCRIPTION
**Pull request by jjlawren**
**Description:**
Allows configuration of the Plex `media_player` platform via config in addition to the nonstandard plex.conf file. Dependent on open PR #10297.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26458

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by DubhAd — DISMISSED** [review]
👍

File: source/_components/plex.markdown [hunk 1]
```diff
@@ -45,57 +48,38 @@ You can also enable the plex platform directly by adding the following lines to
...
 media_player:
   - platform: plex
-```
-
-In the event that [discovery](/components/discovery/) does not work (GDM disabled or non-local Plex server), you can manually create a `plex.conf` file manually and place it in your [configuration directory ](/docs/configuration/) or `/config/` if you are running Hass.io. The following is an example of `plex.conf`:
-
-```json
-{"IP_ADDRESS:PORT": {"token": "TOKEN", "ssl": false, "verify": true}}
+    host: 192.168.1.10
+    use_episode_art: true
```
Comment (frenck) [inline source/_components/plex.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/plex.markdown [hunk 2]
```diff
@@ -45,57 +48,38 @@ You can also enable the plex platform directly by adding the following lines to
...
+    remove_unavailable_clients: true
```
Comment (frenck) [inline source/_components/plex.markdown]: same as above.

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there, @jjlawren, I've reviewed your PR and left some comments. Could you please take a look? Thanks!

**Comment by jjlawren** [comment]
Updated. Can #10297 please be merged before this?

**Review by frenck — APPROVED** [review]
Looks good! Ready to merge 🎉

**Pull request merged**